doc/user_manual/existing_interfaces.tex

```diff
@@ -2352,7 +2352,9 @@ \subsubsection{Files}
 \end{itemize}
 }
 
-Remeber also that a MELCOR simulation run is considered successfull (i.e., the simualtion did not crash) if it terminates with the
+CONTROL FUCTION EDIT data will not be converted in this manner.
```

Comment:
Can you explain a bit better how the CONTROL FUNCTION EDIT are going to be outputted in the CSV file?
For example, something like:
```control_card_$number$```